compiler/rustc_codegen_llvm/src/back/lto.rs

```diff
@@ -39,6 +39,24 @@ pub fn crate_type_allows_lto(crate_type: CrateType) -> bool {
     }
 }
 
+fn get_llvm_reserved_symbols() -> Vec<String> {
```

Comment:
This should be named `get_llvm_preserved_symbols`, not `get_llvm_reserved_symbols`, right?

Reply:
It was my mistake, thanks.